packages/core/src/graph/graph.ts

```diff
@@ -2130,16 +2159,32 @@ export default abstract class AbstractGraph extends EventEmitter implements IAbs
     const nodes: NodeConfig[] = [];
     const edges: EdgeConfig[] = [];
     const combos: ComboConfig[] = [];
+
+    const getModelWithstates = (item: INode | IEdge | ICombo): NodeConfig | EdgeConfig | ComboConfig => {
```

Comment:
This should probably be `getModelWithStates`, but aside from that the PR looks good.

Reply:
yeah. sorry